src/server/middleware/open-api.ts

```diff
@@ -32,19 +31,4 @@ export const withOpenApi = async (server: FastifyInstance) => {
       ],
     },
   });
-
-  await server.register(swaggerUi, {
```

Comment:
if you remove this, the OpenAPI `/json` won't be generated and thus Dashboard won't be able to show the Explore tab